app/retail/templates/mission.html

```diff
@@ -154,8 +162,9 @@ <h4 class="text-left text-uppercase">
           />
 
           <div class="divider"></div>
-
+          {% blocktrans %}
           <p>Intelectual honesty</p>
```

Comment:
Also, is this a duplicate of the above? If so, we might as well drop it.  If not, can we update the spelling here as well?

Reply:
It's a type-o, should have been Collaborative. Fixed.